README.md

```diff
@@ -113,18 +119,40 @@ Ginkgo adds the following additional switches to control what is being built:
 For example, to build everything (in debug mode), use:
 
 ```cmake
-mkdir build; cd build
-cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug -DDEVEL_TOOLS=ON \
-      -DBUILD_TESTS=ON -DBUILD_REFERENCE=ON -DBUILD_OMP=ON -DBUILD_CUDA=ON  ..
-make
+cmake  -G "Unix Makefiles" -H. -BDebug -DCMAKE_BUILD_TYPE=Debug -DGINKGO_DEVEL_TOOLS=ON \
```

Comment:
Is this correct ? You dont need to specify the path to the root `CMakeLists.txt` file ?

Reply:
So, there is no need for a `make` ?

Reply:
The command under what you quoted does the make, `cmake --build Debug`.